common/include/pcl/pcl_macros.h

```diff
@@ -217,7 +217,6 @@
   #endif
 
   #define __PRETTY_FUNCTION__ __FUNCTION__
```

Comment:
```suggestion
  #define __PRETTY_FUNCTION__ __FUNCSIG__
```

See: https://godbolt.org/z/T3P4PWfoj and https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-140

Reply:
Sounds good, thanks